tiledb/sm/query/readers/aggregators/test/unit_aggregators.cc

```diff
@@ -481,17 +481,17 @@ void basic_aggregation_test(std::vector<double> expected_results) {
         10,
         10,
         &fixed_data[0],
-        sizeof(T),
+        sizeof(fixed_data[0]),
```

Comment:
What was the error here?

Reply:
If `T` is `std::string` we need `sizeof(char)` instead of `sizeof(string)`.